RadixWallet/Features/TransactionReviewFeature/TransactionReview.swift

```diff
@@ -789,7 +789,7 @@ extension TransactionReview {
 					public let symbol: String?
 					public let address: ResourceAddress
 					public let icon: URL?
-					public var amount: RETDecimal
+					public var amount: String
```

Comment:
Why? This is not a ViewState, plus the Fungible one has a decimal amount

Reply:
This is mostly about reusing the `poolRedemptionValue` which a returns a string, it will return the special string if we could not calculate the redemption value.